react/components/Input/Input.css

```diff
@@ -1,8 +1,26 @@
 /* removes edge clear button on input */
-.hideDecorators::-ms-clear{
+.hideDecorators::-ms-clear {
   display: none;
 }
 
 .noAppearance {
   appearance: none;
 }
+
+/* placeholders are using c-muted-1's value */
+::-webkit-input-placeholder {
+  /* Chrome/Opera/Safari */
+  color: #727273;
+}
+::-moz-placeholder {
+  /* Firefox 19+ */
+  color: #727273;
+}
+:-ms-input-placeholder {
+  /* IE 10+ */
+  color: #727273;
+}
+:-moz-placeholder {
```

Comment:
You could transform all these rules into a single one: `	::-webkit-input-placeholder, ::-moz-placeholder, :-ms-input-placeholder, :-moz-placeholder { ... }`

Reply:
I should.

Reply:
Does not work:

> This is because a browser will only apply a rule form a selector it can fully interpret. For a webkit type browser -webkit-input-placeholder is valid but -moz-placeholder is not, so it trashes the entire selector, and vise-versa for a geeko based browser. The solution is to separate browser specific selectors.